### DISCUSSION TITLE
Could this be used in WSL?

### DISCUSSION THREAD

**Discussion by jamestansx** [NullSense/fuzzy-sys#3] [Q&A] [answered]
Hi, I am currently using Ubuntu 20.04 WSL version. I am wondering if this could be used in WSL? As far as I know, systemctl isn't used in WSL..

**Comment by NullSense** [comment, marked as answer]
I am not sure if WSL can use systemd, as I don't use windows myself, from some googling it seems there are ways to enable it.
